### PR TITLE
have sf config to argc argv part 5

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

File: tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+#include "sf_c_util/common_argc_argv.h"
+
+#include "sf_c_util/fc_package_com.h"
+#include "sf_c_util/fc_package.h"
+
+static bool argc_argv_are_equal(int argc_left, char** argv_left, int argc_right, char** argv_right)
+{
+    if (argc_left != argc_right)
+    {
+        return false;
```
Comment (dcristoloveanu) [inline tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c:27]: >return false; [](http://example.com/codeflow?start=8&length=13)

Can we have a one return point?

File: tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c [hunk 2]
```diff
@@ -0,0 +1,642 @@
...
+    ///assert(1)
+    ASSERT_IS_NOT_NULL(fc_package);
+    ASSERT_ARE_EQUAL(int, 2, argc_consumed);
+
+    ///act(2)
+    IFabricConfigurationPackage* obj = COM_WRAPPER_CREATE(FC_PACKAGE_HANDLE, IFabricConfigurationPackage, fc_package, fc_package_destroy);
+
+    ///assert(2)
+    ASSERT_IS_NOT_NULL(obj);
+
```
Comment (dcristoloveanu) [inline tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c:98]: This is breaking our own rules about a test testing one thing. The problem with that is "everybody will want one". As in we'll start doing this more and more and we will end up with long tests where we don't really know what is being tested.

So I'd say let's keep ourselves far away from the temptation :-).

File: src/fc_package.c [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+        hr = E_NOT_SET; /*this is NOT_FOUND in HRESULT */
+
+    }
+    return hr;
+}
+
+HRESULT GetValue(FC_PACKAGE_HANDLE fc_package_handle,
+    /* [in] */ LPCWSTR sectionName,
+    /* [in] */ LPCWSTR parameterName,
+    /* [out] */ BOOLEAN* isEncrypted,
```
Comment (dcristoloveanu) [inline src/fc_package.c:195]: >BOOLEAN [](http://example.com/codeflow?start=16&length=7)

Seems windows.h is needed?

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

File: tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c (hunk 1 above)
Comment (anporumb) [reply tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c:27]: We can...

File: src/fc_package.c (hunk 3 above)
Comment (anporumb) [reply src/fc_package.c:195]: added

File: tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c (hunk 2 above)
Comment (anporumb) [reply tests/fc_package_argc_argv_int/fc_package_argc_argv_int.c:98]: Alright, I will break these tests in multiple other tests.

**Pull request merged**